imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -401,36 +427,37 @@ def exchange_rib2mod(self) -> None:
 
     def exchange_sprinkling_demand_msw2rib(self, delt: float) -> None:
         # flux demand from metaswap sprinkling to Ribasim (demand)
-        if self.enable_sprinkling_surface_water:
-            self.msw_sprinkling_demand_sec = (
-                self.msw.get_surfacewater_sprinking_demand_ptr() / days_to_seconds(delt)
-            )
+        self.msw_sprinkling_demand_sec = (
+            self.msw.get_surfacewater_sprinking_demand_ptr() / days_to_seconds(delt)
+        )
+        assert "sw_sprinkling" in self.mapping.msw2rib
```

Comment:
Perhaps better to move this assert to the mapping initialization, since the mappings itself is static?